lib/puppet/parser/functions/human_time_to_seconds.rb

```diff
@@ -0,0 +1,26 @@
+module Puppet::Parser::Functions
+  newfunction(
+    :human_time_to_seconds,
```

Comment:
The problem here is that this file will "fight" with the monitoring_check one.

You will have to namespace this function, how about `cron_human_time_to_seconds`?

In general I'm ok with "vendoring" this function, but you will have to give it a new name.